crates/ibc/src/mock/header.rs

```diff
@@ -71,17 +71,15 @@ impl MockHeader {
     }
 
     pub fn new(height: Height) -> Self {
-        cfg_if::cfg_if! {
```

Comment:
We can probably remove the `cfg_if` dependency (assuming this was the last use?)

Reply:
Ah, sounds like got back b/c of my latest merge from the main
[4b540bc](https://github.com/cosmos/ibc-rs/pull/700/commits/4b540bc3173dd4d1cf33237274da7dc1dd7cd7fe)